packages/document/package.json

```diff
@@ -14,9 +14,5 @@
   "license": "Apache-2.0",
   "publishConfig": {
     "access": "public"
-  },
```

Comment:
Do we still need to keep this in order for Lerna to symlink `tsc` into the package?

If I do a `lerna clean` before `lerna bootstrap --hoist`, no _node_modules_ gets created, and `npm run build` balks.

Reply:
Hmm, lemme take a peek. This *should* work, since I've been doing this multiple times to test